src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
+    // Intentional errors added below
```

Comment:
5. Unit testing: It's important to have unit tests in place for each method to ensure they work correctly under various scenarios, including edge cases like dividing by zero.